test/ci/test-fsim-wget.sh

```diff
@@ -68,35 +70,68 @@ run_test () {
   echo "⭐ Wait for the services to be ready:"
   wait_for_services_ready
 
+  echo "⭐ Prepare the wget test payload file on server side: '${wget_source_file}'"
+  prepare_payload "${wget_source_file}"
+
   echo "⭐ Setting or updating Rendezvous Info (RendezvousInfo)"
   set_or_update_rendezvous_info "${manufacturer_url}" "${rendezvous_service_name}" "${rendezvous_dns}" "${rendezvous_port}"
 
-  echo "⭐ Run Device Initialization"
+  echo "⭐ Run Device Initialization for Device 1"
   run_device_initialization
 
   guid=$(get_device_guid ${device_credentials})
-  echo "⭐ Device initialized with GUID: ${guid}"
+  echo "⭐ Device 1 initialized with GUID: ${guid}"
 
-  echo "⭐ Sending Ownership Voucher to the Owner"
+  echo "⭐ Sending Device 1 Ownership Voucher to the Owner"
   send_manufacturer_ov_to_owner "${manufacturer_url}" "${guid}" "${owner_url}"
 
   echo "⭐ Setting or updating Owner Redirect Info (RVTO2Addr)"
   set_or_update_owner_redirect_info "${owner_url}" "${owner_service_name}" "${owner_dns}" "${owner_port}"
 
-  echo "⭐ Triggering TO0 on Owner server"
+  echo "⭐ Triggering TO0 on Owner server for Device 1 ${guid}"
   run_to0 ${owner_url} "${guid}" > /dev/null
 
-  echo "⭐ Prepare the wget payload on server side: '${wget_test_file}'"
-  prepare_payload "${wget_test_file}"
+  echo "⭐ Running FIDO Device Onboard for Device 1 with FSIM fdo.wget"
+  run_fido_device_onboard --debug --wget-dir "${wget_device_download_dir}"
+
+  echo "⭐ Verify downloaded file: server: ${wget_source_file} device: ${wget_device_output_file}"
+  verify_equal_files "${wget_source_file}" "${wget_device_output_file}"
+  rm -f "${wget_device_output_file}"  # clean up result before next onboarding test
```

Comment:
I am thinking that instead of removing the output file here we could set different output directories per device. This way we don't need to remove anything. WDYT?